fix_laser_sampling _errors.py

```diff
@@ -0,0 +1,75 @@
+#How to change value in hf5 File due to Laser sampling error (high sampling rate)
+
+import tables
+import numpy as np
+import easygui
+import os
+
+# Ask for the directory where the hdf5 file sits, and change to that directory
+dir_name = easygui.diropenbox()
+os.chdir(dir_name)
+
+# Get the names of all files in the current directory, and find the hdf5 (.h5) file
+file_list = os.listdir('./')
+hdf5_name = ''
+for files in file_list:
+	if files[-2:] == 'h5':
+		hdf5_name = files
+
+# Open up the hdf5 file
+hf5 = tables.open_file(hdf5_name, 'r+')
+
+trains_dig_in = hf5.list_nodes('/spike_trains')
+
+# Ask the user for the correct laser duration and convert to integers
+num_durations = easygui.multenterbox(msg = 'how many laser durations were used in this experiment?', fields = ['Number of Laser durations'])
+num_durations = int(num_durations[0])
+
+durations = easygui.multenterbox(msg = 'What are the laser durations used in this experiment?', fields = ["Duration" for num in range(num_durations)])

```

Comment:
Two changes here:
1. You should say in the multenterbox message that you are adding the 0 duration condition automatically (so that people don't choose 2 durations, thinking that 0 is one of them).
2. Change the line slightly to this:
```
durations = easygui.multenterbox(msg = 'What are the laser durations used in this experiment?', fields = ["Duration{}".format(num + 1) for num in range(num_durations)])
```
The box just looks nicer then :P Not very important, but still!